src/navigators/AbstractChildWindow.ts

```diff
@@ -61,6 +61,9 @@ export abstract class AbstractChildWindow implements IWindow {
             };
             window.addEventListener("message", listener, false);
             this._disposeHandlers.add(() => window.removeEventListener("message", listener, false));
+            const channel = new BroadcastChannel(`oidc-client-popup-${params.state}`);
```

Comment:
`AbstractChildWindow` is also used for `IFrameWindow`, i guess there we do not need the `BroadcastChannel`...

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/API/Window/crossOriginIsolated an iframe can be similarly isolated by the Cross-Origin-Embedder-Policy header so the BroadcastChannel could be needed in that case as well. If so then most of the new notifyOpener() logic in PopupWindow (except for window.close()) should be moved up to _notifyParent() in AbstractChildWindow. That seems easier and more future-proof than moving this code down into PopupWindow.

Reply:
> If so then most of the new notifyOpener() logic in PopupWindow (except for window.close()) should be moved up to _notifyParent() in AbstractChildWindow. That seems easier and more future-proof than moving this code down into PopupWindow.

Makes sense!

Reply:
Done.